### DISCUSSION TITLE
Consolidated category feed with multiple listings

### DISCUSSION THREAD

**Discussion by cgoo4** [quarto-dev/quarto-cli#3403] [Q&A] [answered]
I'm setting up a website with two listing pages: _Project_ and _Blog_. If I set up a category feed in both then I get two `index-R.xml` pages. Is there a way to get one consolidated feed for the category?

If I set `listing:` with a feed in the root` index.qmd` (i.e. the one for the landing page), then I'd want to suppress the display of the listing.

```
feed: 
  categories: R
```

**Comment by dragonstyle** [comment, marked as answer]
Right now I can't think of a way - we currently tie the feed to the listing (rather than the category globally), so you'd have to make a listing that contains all the items and then mark category feeds for that (which as you suggest would involve a listing all types of content, which sounds undesirable in your case...).
